apps/opengraph/src/app/share/round/[chainId]/[roundId]/opengraph-image.tsx

```diff
@@ -64,7 +61,7 @@ export default async function Image(props: {
           </div>
           <div tw="flex flex-col items-center">
             <div tw="text-[56px] text-gray-900 text-center mb-2">
-              {round.name}
+              {round.roundMetadata.name}
```

Comment:
This doesn't feel clean.
If we transform it and folks use their own data source, they could just write up their own data transform function and it works